src/styles/components/_label.scss

```diff
@@ -65,6 +65,10 @@ $dso-label-danger-button-hover-color: $dso-danger-color;
       background-color: $dso-label-default-button-hover-background-color;
       border-left-color: $dso-label-default-button-hover-border-color;
       color: $dso-label-default-button-hover-color;
+    }
+
+    &:hover,
+    &:active {
```

Comment:
Dit hoeft niet. Browsers doen niet standaard bij `:hover` of `:active` een outline tekenen, dus die hoeven we ook niet te verwijderen.